### DISCUSSION TITLE
exclude framework

### DISCUSSION THREAD

**Discussion by ChrisSantiago82** [spatie/laravel-backup#1325] [Ideas] [answered]
I'm using this great package for a long time. I have a server that once in a while the backup fails with "Can't open file: No such file or directory"
I have excluded now the  storage/framework directory and since then it's working without any issues anymore. I'm guessing it had to do something the the sessions data.
My question is, should this directory not be excluded by default in the config file?

**Comment by freekmurze** [comment, marked as answer]
Yes, I'll change this in the next major version, as right now it would be breaking change.

**Comment by ChrisSantiago82** [comment]
Ok, perfect. Thanks a lot for this great package.
